src/vivarium_testing_utils/automated_validation/data_transformation/age_groups.py

```diff
@@ -316,12 +314,19 @@ def from_dataframe(cls, df: pd.DataFrame) -> AgeSchema:
         -------
             An AgeSchema with the specified age groups.
         """
-        has_names = AGE_GROUP_COLUMN in df.index.names
-        has_ranges = AGE_START_COLUMN in df.index.names and AGE_END_COLUMN in df.index.names
+        has_names = GBD_INDEX_NAMES.AGE_GROUP in df.index.names
+        has_ranges = (
```

Comment:
These variable names are confusing. I'd suggest `has_age_group` and `has_age_range` or something similar.